lib/models/CartoonCat.js

```diff
@@ -1,4 +1,25 @@
 const pool = require('../utils/pool');
 module.exports = class CartoonCat {
-  // TODO -- fill in this class!
+  constructor(row) {
+    this.id = row.id;
+    this.name = row.name;
+    this.type = row.type;
+    this.lives = row.lives;
+    this.url = row.url;
+    this.year = row.year;
+    this.isSidekick = row.is_sidekick;
+  }
+
+  static async getAll() {
+    const { rows } = await pool.query('SELECT * from cartoon_cats');
+    return rows.map((catRow) => new CartoonCat(catRow));
+  }
+
+  static async getById(id) {
+    const { rows } = await pool.query(
+      'SELECT * from cartoon_cats WHERE id = $1',
```

Comment:
generally the convention is all caps for SQL outside of table names / column names (to help differentiate between SQL and table names visually), or no caps at all, so either:
```SELECT * FROM cartoon_cats WHERE id = $1```
or
```select * from cartoon_cats where id = $1```